### PR TITLE
Package Cascadia Code with the Terminal (release builds)

### PR DESCRIPTION
**Pull request by DHowett-MSFT**
TTF from microsoft/cascadia-code@5f91b87e12c9af709fa406c22a79145879edc3d2

### CODE REVIEW

**Comment by cinnamon-msft** [comment]
Just to confirm, the font installs on download?

**Comment by DHowett-MSFT** [comment]
The font installs when the package is installed. The font is removed when the package is removed. _When you want to uninstall the font, you must remove the package (!)_

**Comment by DHowett-MSFT** [comment]
For all intents and purposes, the font is the package.

**Comment by cinnamon-msft** [comment]
So if you don't want the font, you have to uninstall the Terminal package?

**Comment by DHowett-MSFT** [comment]
> you have to uninstall

Yes.

**Comment by zadjii-msft** [comment]
Out of curiosity, what happens when I install Cascadia.ttf separately? Does the package install a second copy? If I remove Cascadia.ttf, but leave the package, is the font still installed? If I remove the package but not the ttf, does it stay installed?

**Comment by DHowett-MSFT** [comment]
This one in particular requires @cinnamon-msft to sign off.

File: res/README.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
+Please consult the [license](./LICENSE) in this directory for terms applicable to the image assets in this directory.
+
+## Fonts
+
+The fonts in this directory do not fall under the same [license](https://raw.githubusercontent.com/microsoft/terminal/master/LICENSE) as the rest
 of the Windows Terminal code.
 
-Please consult the [license](./LICENSE) in this directory for applicable terms.
+Please consult the [license](https://raw.githubusercontent.com/microsoft/cascadia-code/master/LICENSE) in the
+[microsoft/cascadia-code](https://github.com/microsoft/cascadia-code) repository for terms applicable to the fonts in this directory.
```
Comment (DHowett-MSFT) [inline res/README.md]:
```suggestion
[microsoft/cascadia-code](https://github.com/microsoft/cascadia-code) repository for terms applicable to the fonts in this directory.

### Font Manifest

* Cascadia Code
   * From microsoft/cascadia-code@5f91b87e12c9af709fa406c22a79145879edc3d2
```

**Comment by cinnamon-msft** [comment]
> Out of curiosity, what happens when I install Cascadia.ttf separately? Does the package install a second copy? If I remove Cascadia.ttf, but leave the package, is the font still installed? If I remove the package but not the ttf, does it stay installed?

Did we test this scenario? I'd assume the TTF would persist but I'm not sure.

**Comment by miniksa** [comment]
I'm concerned that people are already asking for this as a standalone Windows Store package (https://github.com/microsoft/cascadia-code/issues/39) and that bundling it inside the Terminal package would create a conflict with that or multiple entries.

Are we going to distribute it this way "for a limited time" until we figure out how to release it standalone and maybe request it as a dependency/download when you install the Terminal package?

Or do we always intend to ship a copy of it inside the Terminal package?

**Comment by DHowett-MSFT** [comment]
@miniksa this is temporary until we can add a dependency on the official package.
@cinnamon-msft yes, the ttf persists and the terminal can be uninstalled. i _think_ the ttf version wins when both are installed. I'm not sure.

**Pull request merged**